velox/exec/Window.h

```diff
@@ -22,6 +22,16 @@
 
 namespace facebook::velox::exec {
 
+// Structure for k preceding/following frame arguments, index stores the index
+// of the column when k is a column and kConstantChannel for constant k. When k
+// is a column, value points to a FlatVector containing the columnar offsets,
+// when k is a constant integer, value points to a ConstantVector of length 1
+// containing the constant offset.
+struct FrameChannelArg {
+  int64_t index;
```

Comment:
Please use column_index_t type for this.

Reply:
Apologies for missing this. Changed to column_index_t.